### PR TITLE
Document IPYTHON_SUPPRESS_CONFIG_ERRORS.

### LINKED ISSUE #9493: Document IPYTHON_SUPPRESS_ERRORS
See  https://github.com/ipython/ipython/pull/9478/files

### PR DESCRIPTION
**Pull request by Carreau**
Closes #9493

### CODE REVIEW

File: docs/source/development/config.rst [hunk 1]
```diff
@@ -55,6 +55,17 @@ under :file:`profile_default`. If you want the default config files for the
...
 command-line args.
 
+.. note::
+
+    By default IPython will ignore any invalid configuration files, or any
+    unknown configuration option.
+
+.. versionadded:: 5.0
+
+    IPython can be configured to abort in case of invalid configuration file.
```
Comment (takluyver) [inline docs/source/development/config.rst]: I think we should mention that this doesn't catch misnamed keys or invalid values: 'invalid' means we couldn't load it at all.

Comment (Carreau) [inline docs/source/development/config.rst]: That's what I was trying to say, but I got the phrasing wrong.

**Comment by Carreau** [comment]
Is that better ?

**Comment by takluyver** [comment]
Yep, thanks. Not sure why I didn't see this earlier.

**Pull request merged**